test/e2e/tests/onboarding/onboarding.spec.ts

```diff
@@ -309,7 +308,7 @@ describe('MetaMask onboarding', function () {
         title: this.test?.fullTitle(),
       },
       async ({ driver }) => {
-        await loginWithoutBalanceValidation(driver);
+        await unlockWallet(driver);
```

Comment:
here we can't use the method, because we don't land at the Home page screen, but into the Metrics page (expected), so we should use unlockWallet method instead

![test-failure-screenshot-1](https://github.com/user-attachments/assets/e0cb74c6-a6bd-4b7a-80c5-0a860dfc0369)